CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
 # Changelog
+## v3.3.0
```

Comment:
do we need to bump sei-ibc-go in go.mod?

Reply:
It's already bumped in previous PR: https://github.com/sei-protocol/sei-chain/pull/1137